### PR TITLE
possible fix for .png black background

### PR DESCRIPTION
**Pull request by netgfx**
fixes issue #88

### CODE REVIEW

File: android/src/main/java/com/example/flutternativeimage/MethodCallHandlerImpl.java [hunk 1]
```diff
@@ -120,24 +120,27 @@ public void onMethodCall(MethodCall call, MethodChannel.Result result) {
                 result.error("file does not exist", fileName, null);
                 return;
             }
+            Boolean isPNG = fileName.contains(".png") || fileName.contains(".PNG");
```
Comment (btastic) [inline android/src/main/java/com/example/flutternativeimage/MethodCallHandlerImpl.java:123]:
```suggestion
            Boolean isPNG = fileName.toLowerCase().contains(".png");
```

File: android/src/main/java/com/example/flutternativeimage/MethodCallHandlerImpl.java [hunk 2]
```diff
@@ -120,24 +120,27 @@ public void onMethodCall(MethodCall call, MethodChannel.Result result) {
...
+            Bitmap.CompressFormat format = ((isPNG == true) ? Bitmap.CompressFormat.PNG : Bitmap.CompressFormat.JPEG);
```
Comment (btastic) [inline android/src/main/java/com/example/flutternativeimage/MethodCallHandlerImpl.java:124]:
```suggestion
            Bitmap.CompressFormat format = isPNG ? Bitmap.CompressFormat.PNG : Bitmap.CompressFormat.JPEG;
```

File: android/src/main/java/com/example/flutternativeimage/MethodCallHandlerImpl.java [hunk 3]
```diff
@@ -120,24 +120,27 @@ public void onMethodCall(MethodCall call, MethodChannel.Result result) {
...
+            String extension = ((isPNG == true) ? ".png" : ".jpg");
```
Comment (btastic) [inline android/src/main/java/com/example/flutternativeimage/MethodCallHandlerImpl.java:125]:
```suggestion
            String extension = isPNG ? ".png" : ".jpg";
```

File: android/src/main/java/com/example/flutternativeimage/MethodCallHandlerImpl.java [hunk 4]
```diff
@@ -120,24 +120,27 @@ public void onMethodCall(MethodCall call, MethodChannel.Result result) {
...
 
             Bitmap bmp = BitmapFactory.decodeFile(fileName);
             ByteArrayOutputStream bos = new ByteArrayOutputStream();
-
+
```
Comment (btastic) [inline android/src/main/java/com/example/flutternativeimage/MethodCallHandlerImpl.java:129]:
```suggestion

```

**Review by btastic — CHANGES_REQUESTED** [review]
Thanks!

**Comment by netgfx** [comment]
committed all the suggestions (sorry I haven't written Java in more than 10 years :D )

**Comment by btastic** [comment]
No worries! I'm thankful for every line of code :)

Thanks!

**Pull request merged**